webui/master/src/containers/pages/Overview/Overview.tsx

```diff
@@ -134,6 +135,21 @@ export class OverviewPresenter extends React.Component<AllProps> {
     );
   }
 
+  private renderJournalCheckpointWarning(warning: string, className: string): JSX.Element | null {
+    if (!warning || warning == "") {
+      return null;
+    }
+
+    return (
+      <tr key="0">
```

Comment:
make the key something more unique - '0' might collide with `renderJournalDiskWarnings()` table rows